pysat/tests/classes/cls_instrument_library.py

```diff
@@ -66,14 +66,57 @@ def initialize_test_inst_and_date(inst_dict):
     test_inst = pysat.Instrument(inst_module=inst_dict['inst_module'],
                                  tag=inst_dict['tag'],
                                  inst_id=inst_dict['inst_id'],
-                                 temporary_file_list=True,
-                                 update_files=True, use_header=True,
-                                 **kwargs)
+                                 temporary_file_list=True, update_files=True,
+                                 use_header=True, **kwargs)
     test_dates = inst_dict['inst_module']._test_dates
     date = test_dates[inst_dict['inst_id']][inst_dict['tag']]
     return test_inst, date
 
 
+def set_strict_time_flag(test_inst, date, raise_error=False):
+    """Ensure the strict time flag does not interfere with other tests.
```

Comment:
Technically, this loads data in addition to setting the `strict_time_flag`. Tested this code manually, and it does what it should. Modifying `test_inst` in place makes me a little nervous, but it's probably fine as long as the docstring is clear.